### DISCUSSION TITLE
[solved] Background Bug v4.19

### DISCUSSION THREAD

**Discussion by Staini** [AppGround-io/bluetooth-keyboard-and-mouse-support#258] [Bugs & Problems]
The background should normally be black:

![Screenshot_20220923-100737_Bluetooth Keyboard   Mouse~2](https://user-images.githubusercontent.com/91640388/191918072-79d1cb06-0686-4836-8bd7-e0bb0309d9b8.jpg)

**Comment by Staini** [comment]
Thanks for the 4.19.1 fix!

**Reply by AppGround-io** [reply]
Thank you for reporting the bug!
